stko/utilities/utilities.py

```diff
@@ -1127,3 +1128,81 @@ def calculate_dihedral(pt1, pt2, pt3, pt4):
     x = np.dot(v, w)
     y = np.dot(np.cross(b1, v), w)
     return np.degrees(np.arctan2(y, x))
+
+
+def get_torsion_info_angles(mol: stk.ConstructedMolecule,
```

Comment:
Reformat and have complete type hints (I am unsure if the below example requires importing TorsionInfo, which we do not want.):

```python
def get_torsion_info_angles(
    mol: stk.ConstructedMolecule,
    torsion_info: Type['TorsionInfo'],
):

```

Reply:
This should be 'TorsionInfo' not Type[]

Reply:
Yes, I think I got confused as well - still learning about type hints. I would say (and have commented in the review) to remove all type hints for now.